packages/google_maps_flutter/ios/Classes/GoogleMapController.m

```diff
@@ -81,10 +82,17 @@ - (instancetype)initWithFrame:(CGRect)frame
     _markersController = [[FLTMarkersController alloc] init:_channel
                                                     mapView:_mapView
                                                   registrar:registrar];
+    _polylinesController = [[FLTPolylinesController alloc] init:_channel
+                                                        mapView:_mapView
+                                                      registrar:registrar];
     id markersToAdd = args[@"markersToAdd"];
     if ([markersToAdd isKindOfClass:[NSArray class]]) {
       [_markersController addMarkers:markersToAdd];
     }
+    id polylinesToAdd = args[@"polylinesToAdd"];
```

Comment:
nit: this is my personal opinion, feel free to ignore. Declare as `id` is not helpful to understand what `args[@"polylinesToAdd"]` should be expected to be. I had to look the code below to see that you are checking the type to know that you are expecting an NSArray. You would still need the type check below, but at least by reading this line, I know what to expect from the `args[@"polylinesToAdd"]`.
```suggestion
    NSArray* polylinesToAdd = args[@"polylinesToAdd"];
```
Same as below, from line 136 to line 149

Reply:
I tried to consistent with the existing logic to add markers. For example, in line  88 `markersToAdd` is also stored as id. I would love to change polyline one to the NSArray but not sure what to do with the marker. Can you suggest how should I move on?